### DISCUSSION TITLE
Do I have an erroneous response from the suprabase endpoint?  I don't see the data property  👋

### DISCUSSION THREAD

**Discussion by NakedSnakeMG3** [supabase/supabase#11623] [Questions]
when I make a request using the documentation generated by suprabase the data property of the response does not exist, only body. status, statusCode and statusText. Does anyone have the same problem?

`{
    "body": [
        {
            "id": 1,
            "inserted_at": "2023-01-12T22:05:10.848643+00:00",
            "updated_at": "2023-01-12T22:05:10.848643+00:00",
            "data": null,
            "name": "haru"
        }
    ],
    "status": 200,
    "statusCode": 200,
    "statusText": ""
}`

**Comment by GaryAustin1** [comment]
Not at all clear what you are doing as you don't show code.
But insert does not return data unless you add .select() to the request (supabase-js)

**Reply by NakedSnakeMG3** [reply]
let me add some code
this is the code that of API Code Supabase suggest to fetch the prducts
```
let { data: products, error } = await supabase
  .from('products')
  .select('*')
```
but when I view the logs the request returns the following datathanks for your time. 
![foto](https://user-images.githubusercontent.com/31257678/212359717-bede446a-95e8-4fbe-b414-54a354852cea.png)

This is my CODE

```
export default function AboutPage() {
    const [products, setProducts] = useState();
    useEffect(() => {
        const getProducts = async () => {
            let data = await supabaseClient
                .from('products')
                .select('*');
            console.log(data, 'getProducts');
            setProducts(data);

        }
        getProducts();
    }, []);
    return (<h1>this is about</h1>);
}
```

thanks for your time :)

**Reply by GaryAustin1** [reply]
That is odd.
It looks like your are getting the raw fetch data back directly.
This code is supposed to take what you are showing in the response and convert to the supabase format...
https://github.com/supabase/postgrest-js/blob/master/src/PostgrestBuilder.ts#L70

You can see it takes the response body and moves it to the data object and also sets up count, error, etc.
For some reason that code is not running.
